epub_translator/epub2/html/empty_tags.py

```
@@ -0,0 +1,23 @@
+import re
+
+# HTML 规定了一系列自闭标签，这些标签需要改成非自闭的，因为 EPub 格式不支持
```

Ruff (0.14.10)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)